percona-xtradb-cluster/5.5/source/release-notes/Percona-XtraDB-Cluster-5.5.41-25.11.rst

```diff
@@ -0,0 +1,67 @@
+.. rn:: 5.5.41-25.11
+
+=======================================
+ |Percona XtraDB Cluster| 5.5.41-25.11
+=======================================
+
+Percona is glad to announce the release of |Percona XtraDB Cluster| on March 30th, 2015. Binaries are available from `downloads area <http://www.percona.com/downloads/Percona-XtraDB-Cluster/5.5.41-25.11/>`_ or from our :doc:`software repositories </installation>`.
+
+Based on `Percona Server 5.5.41-37.0 <http://www.percona.com/doc/percona-server/5.5/release-notes/Percona-Server-5.5.41-37.0.html>`_ including all the bug fixes in it, Galera Replicator 2.11 and on wsrep API `25.11 <https://launchpad.net/codership-mysql/+milestone/5.5.38-25.11>`_), |Percona XtraDB Cluster| `5.5.41-25.11 <https://launchpad.net/percona-xtradb-cluster/+milestone/5.5.41-25.11>`_ is now the current stable release. All of |Percona|'s software is open-source and free. 
+
+This is an General Availability release. We did our best to eliminate bugs and problems during the testing release, but this is a software, so bugs are expected. If you encounter them, please report them to our `bug tracking system <https://bugs.launchpad.net/percona-xtradb-cluster/+filebug>`_.
+
+
+New Features
+============
+
+ |Percona XtraDB Cluster| 5.5.41-25.11 packages are now available for *Ubuntu* 14.10 (utopic).  
```

Comment:
No need to mention this under 'New Features'. Either mention in section above or skip it.